pkg/apis/stork/v1beta1/doc.go

```diff
@@ -0,0 +1,5 @@
+// +k8s:deepcopy-gen=package,register
```

Comment:
Why is this beta1? Let's use the convention of starting these as alpha1, we can update them all later when it has gone through some releases.

Reply:
v1beta1 should not be present in this PR. Removed the pkg.